README.md

```diff
@@ -1,66 +1,66 @@
-Yii PHP Framework Version 2
+Yii PHP Framework Version 2 [![Total Downloads](https://poser.pugx.org/yiisoft/yii2/downloads.png)](https://packagist.org/packages/yiisoft/yii2) [![Latest Stable Version](https://poser.pugx.org/yiisoft/yii2/v/stable.png)](https://packagist.org/packages/yiisoft/yii2)
 ===========================
 
-Thank you for choosing Yii 2 - a modern PHP framework designed for professional Web development.
+Thank you for choosing Yii2 - a modern PHP framework designed for professional Web development.
+Yii2 inherits the main spirit behind Yii for being highly extensible, secure and fast.
+We strive to embrace all the best practices applied in modern Web application development.
```

Comment:
I'd remove "all" because it's not true :) We're skipping some for the sake of API...